src/test_l1_dsAudio.c

```diff
@@ -75,15 +75,31 @@
 #include "dsAudioSettings.h"
 #include <ut.h>
 #include <ut_log.h>
+#include <ut_kvp_profile.h>
 
 #define NUM_OF_PORTS (sizeof(kPorts) / sizeof(kPorts[0]))
 #define MAX_PROFILE_NAME_LEN 10
 #define INT_ARRAY_INIT 0
 
+#define CHECK_FOR_EXTENDED_ERROR_CODE( result, enhanced, old )\
```

Comment:
#defines have GLOBAL Scope, you'll be re-using the same macro in all files..

therefore has to be module specific

`DS_AUDIO_CHECK_FOR_EXTEND_ERROR_CODE`

it's only working by luck.